.changeset/poor-bugs-reflect.md

```diff
@@ -0,0 +1,47 @@
+---
+'@xstate/graph': minor
+---
+
+Options passed into graph functions (e.g., `getShortestPaths(machine, options)`) can now resolve `.events` based on the `state`:
```

Comment:
Just a praise comment for Changesets idea 🙇 

Super useful for readers with a descriptive comment and example!